dark-mode.css

```
@@ -927,7 +927,7 @@ body.dark-mode .main-content  *::-webkit-scrollbar-corner {
 }
 
 /***** Poll App *****/
-body.dark-mode [data-username="poll.bot"] .rcx-css-erwtrf {
+body.dark-mode [data-username="app.poll"] .rcx-css-erwtrf {
```

```suggestion
body.dark-mode [data-username="poll.bot"] .rcx-css-erwtrf, 
body.dark-mode [data-username="app.poll"] .rcx-css-erwtrf {
```
`poll.bot` is still required for 3.13.x but 3.14 uses `app.poll`, so let's support both for now.

---

Hey, Paul!
I just tested more universal solution - just use class `.rcx-css-erwtrf` without `data-username` at all

---

This looks good to me! Hopefully it doesn't break other apps display. Merging.